lib/resources/postgres.rb

```diff
@@ -81,6 +77,16 @@ def verify_dirs
       end
     end
 
+    def version_from_psql
```

Comment:
I don't think we need the warn here. I'd recommend we simplify this to:

```
return unless inspec.command('psql').exist?
inspec.command("psql --version | awk '{ print $NF }' | awk -F. '{ print $1\".\"$2 }'").stdout.strip
```

Ultimately if we need users to know how we determined how we found the version, we can create a method they can assert a value on... i.e.:

```
describe postgres do
  its('detection_mechanism') { should eq 'psql' }
  its('detection_mechanism') { should eq 'versioned_data_directory' }
  its('detection_mechanism') { should eq 'unversioned_data_directory' }
end
```

I think the warnings clutter up the output too much, and for users using a structured formatter (i.e. junit, json, etc.), they have to worry about redirecting stderr out of the way.